Lesson5/Task2/Car/Calculator.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Task2
+{
+    class Calculator
+    {
+        private int summary;
```

Comment:
@okuzmina Why we need such field in the whole class, not just in method Summary ?
It could be useful if another parts of the class (properties or methods) will use it, but for now it could be only var in the Summary method